public/javascript/posts/BookAppointment.js

```diff
@@ -220,10 +245,35 @@ var app = angular.module('BookAppointmentApp');
         });
       };
 
+      $scope.sendDoctorInfo = function(){
+        $scope.extend($scope.doctor_event_info, $scope.omit($scope.selected_event, 'jsEvent', 'view'));
+
+        $http.post('/create_doc',$scope.doctor_event_info).success(function(response){
+          $scope.addEvent();

```

Comment:
create a function called as book appointment will take two parameters , event_hash and url_to_post , in this method write logic - 
$http.post(url_to_post, event_hash).success(function(response){
          $scope.addEvent();
          $scope.appointmentBooked();
        });